jekyll/_cci2/github-app-trigger-options.adoc

```diff
@@ -0,0 +1,83 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= GitHub App trigger options
```

Comment:
should this be "trigger event options"?

Reply:
Cheeky ask here, but somewhere in this or other pages, could we get a header with "run pipelines non non-push events", for SEO?